eval/safe/config.py

```diff
@@ -34,7 +34,7 @@
 #     'claude_instant',
 # ]
 ################################################################################
-model_short = 'gpt_35_turbo'
+model_short = 'gpt_4o_mini'
```

Comment:
Probably we should keep the default to be gpt-3.5-turbo unless we've verified gpt-4o-mini works on par or better.

Reply:
Reverted to GPT-3.5 Turbo